src/TorchVision/AutoAugment.cs

```diff
@@ -9,7 +9,7 @@ namespace TorchSharp
 {
     public static partial class torchvision
     {
-        internal abstract class AutoAugmentBase
+        public abstract class autoaugment
```

Comment:
Why the lower-case class name? Even PyTorch uses capitalization on class names.

Reply:
Its a namespace hack, because this class is exposed publicly to give: `torchvision.autoaugment.AutoAugmentPolicy` which matches the convention of PyTorch. There is no AutoAugmentBase class in pytorch. Maybe I can make the base internal again, and use a namespace instead.